prometheus/plan.sh

```diff
@@ -38,7 +37,7 @@ do_unpack() {
 
 do_build() {
   pushd "$prom_pkg_dir/src/github.com/prometheus/prometheus" || exit 1
-  PREFIX="$pkg_prefix/bin" make build
+  USER="root" PREFIX="$pkg_prefix/bin" make build
```

Comment:
Is this the cgo change you referenced in your PR?  Would you mind leaving a couple words in the PR what was broken and how this change fixed it?  Or if it's a cgo-ism,  are there any docs we could link to?